data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json

```
@@ -75,5 +75,62 @@
     "min_range": 4,
     "max_range": 8,
     "range_increment": 0.12
+  },
+  {
+    "id": "cleansingflame_hermes_heal",
+    "type": "SPELL",
+    "spell_class": "CLEANSING_FLAME",
+    "name": "Hermes reborn",
+    "description": "In the sea without lees \nStandeth the bird of Hermes \nEating his wings variable \nAnd maketh himself yet full stable",
```

```suggestion
    "description": "In the sea without lees\nStandeth the bird of Hermes\nEating his wings variable\nAnd maketh himself yet full stable",
```
Is this from someplace, BTW?

---

It is from The Ripley Scroll. A Western esoteric piece